best-practices.md

```diff
@@ -72,13 +91,18 @@ is to place the catalog file in namespaces "directories". For example:
 
 ### Dynamic Catalogs
 
-Dynamic STAC Catalogs are those that generate their JSON responses programmatically instead of relying on a set of
-already defined files. Typically a dynamic catalog implements the full [STAC API](https://github.com/radiantearth/stac-api-spec) which enables 
-search of the Items indexed. The `/` endpoint returns the exact same STAC Catalog structure as a
-static catalog, enabling the same discovery from people browsing and search engines crawling. Dynamic API's that
-just seek to expose some data can also choose to not implement `/search` and only link to their data from the `/` endpoint.
-For example a Content Management Service like Drupal or an Open Data Catalog like CKAN could choose to expose its content
-as linked STAC Items by implementing a dynamic catalog. 
+A dynamic catalog is implemented in software as a RESTful API, following the same specified JSON structure for Items, Catalogs
```

Comment:
We do HTTP APIs, it's not strictly RESTful, because we have some pre-defined paths...

Reply:
Fair, I can tone it down.